### PR TITLE
Create separate tab for adequacy patch options

### PR DESCRIPTION
**Pull request by Milos-RTEi**

### CODE REVIEW

**Comment by Milos-RTEi** [comment]
Hi! Take a look of what I have done so far, if you have a chance. 
I completed and grouped the adq. options/buttons in one window (and *.ini file and code). 
I will start the area data in the separate window first, and join them later if necessary.

**Comment by Milos-RTEi** [comment]
Hi guys! i got stuck and need some help with c++ debugging.

If you pull this pr and try to build antares you will get an error: "/src/ui/simulator/windows/options/adequacy-patch/adequacy-patch-areas.cpp:70: undefined reference to `Antares::Component::Datagrid::Renderer::AdequacyPatchAreaGrid::AdequacyPatchAreaGrid()"

I clearly created AdequacyPatchAreaGrid() class in new *.h and *.cpp files and linked them properly with #include "paths".
Spent hours on google/forums trying to figure out what is the problem. It's a very common problem and lots of proposed solutions but nothing worked. As usual I just followed the same procedure as for some other similar 'window" where this works fine, and it does not work here,
 Please take a look. When this is resolved rest of the solution should work just fine.

File: src/libs/antares/study/parameters.cpp [hunk 1]
```diff
@@ -620,6 +659,37 @@ static bool SGDIntLoadFamily_Optimization(Parameters& d,
...
+        return value.to<bool>(d.setToZero12LinksForAdequacyPatch);
+    if (key == "set-to-null-ntc-between-physical-out-for-first-step")
+        return value.to<bool>(d.setToZero11LinksForAdequacyPatch);
+    if (key == "save-intermediate-results")
+        return value.to<bool>(d.adqPatchSaveIntermediateResults);
+    // Price taking order
+    if (key == "price-taking-order")
+    {
+        return StringToPriceTakingOrder(value, d.adqPatchPriceTakingOrder);
+        return false;
```
Comment (rtei-pw) [inline src/libs/antares/study/parameters.cpp]: ?

**Review by rtei-pw — CHANGES_REQUESTED** [review]

File: src/libs/antares/study/parameters.cpp (hunk 1 above)
Comment (Milos-RTEi) [reply src/libs/antares/study/parameters.cpp]: Corrected.

**Pull request merged**